documentation/book/appendix_metrics.adoc

```diff
@@ -41,10 +41,26 @@ https://prometheus.io/[Prometheus] is an open-source set of components for syste
 
 To deploy the Prometheus Operator in your Kafka cluster, apply the `.yaml` files from the https://github.com/coreos/prometheus-operator/tree/master/example/rbac/prometheus-operator[Prometheus CoreOS repository].
 
-To use a different namespace than that specified in the repository files, use the following command to download and edit the `prometheus-operator-deployment.yaml` file:
+To use a different namespace than that specified in the repository files, use the following commands to download and edit the files from respository:
 
 [source,shell,subs="+quotes"]  
 curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/rbac/prometheus-operator/prometheus-operator-deployment.yaml | sed -e 's/namespace: .*/namespace: _my-namespace_/;s/regex: myproject/regex: _my-namespace_/' > prometheus-operator-deployment.yaml
+curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/rbac/prometheus-operator/prometheus-operator-cluster-role-binding.yaml | sed -e 's/namespace: .*/namespace: _my-namespace_/;s/regex: myproject/regex: _my-namespace_/' > prometheus-operator-cluster-role-binding.yaml
+curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/rbac/prometheus-operator/prometheus-operator-service-account.yaml | sed -e 's/namespace: .*/namespace: _my-namespace_/;s/regex: myproject/regex: _my-namespace_/' > prometheus-operator-service-account.yaml
+curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/rbac/prometheus-operator/prometheus-operator-cluster-role.yaml
+
+It might happen you need to remove `securityContext` from Prometheus Operator Deployment which is located in the `prometheus-operator-deployment.yaml` file.
```

Comment:
Well it should be true for OpenShift only I guess, so we should highlight this. The bigger question is, did you test this stuff on minikube?

Reply:
I did. I also had some issue but I do not recall what that was.

Reply:
What do you mean by "issue"? Did it work? Do we need a "workaround" for minikube as well?

Reply:
I am not sure :O but I think it did not work. Let me try it again.

Reply:
If it didn't work we have to understand why :-) Otherwise the PR will be not ready to be merged